### PR TITLE
init style battle

### PR DESCRIPTION
**Pull request by Swiip**
capture d ecran 2018-12-31 a 10 05 33

Initiate loading pokemon stats.

Wondering which logic framework to use for combat

### CODE REVIEW

**Comment by mfrachet** [comment]
This is AWESOME ! 😍

File: src/components/arena-content.js [hunk 1]
```diff
@@ -1,19 +1,135 @@
...
+          <LifeBarContainer>
+            <LifeBar health={getHp(data[0]) / getHp(data[0])} />
+          </LifeBarContainer>
+          <LifeBarNumbers>
+            {getHp(data[0])} / {getHp(data[0])}
+          </LifeBarNumbers>
+        </LifeBarSection>
+      </StatBarFirst>
+      <PokemonSecond src={`http://localhost:4200/${second}.gif`} alt={second} />
+      <StatBarSecond>
```
Comment (mfrachet) [inline src/components/arena-content.js]: From l121 to l131 can also be moved somewhere in a specific component with a props like `position="{front|back}"`

But never mind, if we have the time later 😊

**Review by mfrachet — APPROVED** [review]

File: src/components/arena-content.js (hunk 1 above)
Comment (Swiip) [reply src/components/arena-content.js]: Sure!

**Comment by Swiip** [comment]
capture d ecran 2019-01-02 a 12 08 37

If you have a doubt... I'm pretty proud of this ^^

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,3 @@
 node_modules/
 api/data/
+.DS_Store
```
Comment (FBerthelot) [inline .gitignore]: a mettre dans ton .gitignore global :)

File: src/components/arena/oponent.js [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+  border: 2px solid black;
+  border-radius: 3px;
+  height: 20px;
+  width: 200px;
+`;
+
+const getColor = health => {
+  if (health > 0.5) {
+    return "lime";
+  } else if (health > 0.2) {
```
Comment (FBerthelot) [inline src/components/arena/oponent.js]:
```suggestion
  }
  if (health > 0.2) {
```

File: src/components/arena/oponent.js [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+    return "orange";
+  } else {
```
Comment (FBerthelot) [inline src/components/arena/oponent.js]:
```suggestion
  }
```

File: src/components/arena/oponent.js [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+    return "red";
+  }
```
Comment (FBerthelot) [inline src/components/arena/oponent.js:61]:
```suggestion
```

File: src/components/arena/oponent.js [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+  width: 200px;
+`;
+
+const getColor = health => {
+  if (health > 0.5) {
+    return "lime";
+  } else if (health > 0.2) {
+    return "orange";
+  } else {
+    return "red";
```
Comment (FBerthelot) [inline src/components/arena/oponent.js]:
```suggestion
  return "red";
```

**Review by FBerthelot — APPROVED** [review]

File: .gitignore (hunk 2 above)
Comment (Swiip) [reply .gitignore]: j'y avais jamais pensé ;) c'est fait :p

**Pull request merged**